### PR TITLE
CNV-45152: Added note about linux bonding mode support

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.14+

Issue: [CNV-45152](https://issues.redhat.com//browse/CNV-45152)

Link to docs preview: https://85882--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-connecting-vm-to-linux-bridge.html

https://85882--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.html#virt-creating-localnet-nad-cli_virt-connecting-vm-to-ovn-secondary-network

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc [hunk 1]
```diff
@@ -14,6 +14,11 @@ You can create a Linux bridge network and attach a virtual machine (VM) to the n
 . Create a Linux bridge network attachment definition (NAD) by using the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-creating-linux-bridge-nad-web_virt-connecting-vm-to-linux-bridge[web console] or the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-creating-linux-bridge-nad-cli_virt-connecting-vm-to-linux-bridge[command line].
 . Configure the VM to recognize the NAD by using the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-vm-creating-nic-web_virt-connecting-vm-to-linux-bridge[web console] or the xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-attaching-vm-secondary-network-cli_virt-connecting-vm-to-linux-bridge[command line].
 
+[NOTE]
+====
+{VirtProductName} does not support Linux bridge bonding modes 5 and 6. For more information, see link:https://access.redhat.com/solutions/67546[Which bonding modes work when used with a bridge that virtual machine guests or containers connect to?].
```
[the remark this answers is not in the document]
Comment (phoracek) [reply virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc:19]: If RHV did not support it, we should not either.

**Review by phoracek — CHANGES_REQUESTED** [review]

**Review by phoracek — APPROVED** [review]

**Review by lahinson — COMMENTED** [review]
LGTM

**Review by yossisegev — APPROVED** [review]
/lgtm

**Pull request merged**